test/linter/test.sh

```diff
@@ -80,6 +80,11 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+helm lint . --strict --set registry.initContainers.dummy.image=foo
```

Comment:
We need test case both for registry and sync. And also positive and negative case for both: positive when passing list of object as value and negative when passing schema violating value.